assets/css/style.css

```diff
@@ -441,6 +440,210 @@ input {
   cursor: pointer;
 }
 
+/* Start Modal */
+.modal {
+  width: 100%;
+  height: 100%;
```

Comment:
- [ ] Great job working on this milestone. I have to scroll down the pop up window card before seeing the See Live and See Source buttons. Kindly adjust the height of the modal card in both desktop and mobile views to fit on the screen. See screenshots below
- Your desktop design
![image](https://github.com/WilfriedM01/portfolio/assets/62849872/8db84e28-3073-4be3-8646-6adcce176624)

- Your mobile design
![image](https://github.com/WilfriedM01/portfolio/assets/62849872/f8153edf-c45f-4e1b-8656-44fdb3020b35)

Reply:
Thank you @t-yanick , 
I updated and ran the Figma design template1, and it looks the same.  Thank you